zephyr/README.md

```diff
@@ -0,0 +1,3 @@
+The OpenThread stack is integrated with ZephyrOS and nRF Connect SDK.
```

Comment:
Somehow my comment was lost by GitHub... Could you please move this `README.md` to `examples/platforms/zephyr` to align with other platforms?